lib/git_local/repository.rb

```diff
@@ -60,6 +64,13 @@ def path
       @path ||= "#{local_directory}/#{org_repo_branch}"
     end
 
+    def check_for_special_characters(*args)
+      regexp = Regexp.new(/([A-Za-z0-9\-\_\/]+)/)
```

Comment:
Mind including a test for this behavior? Also, will this break with our `foo#123` branch syntax?